docs/release_notes/10_0.md

```diff
@@ -0,0 +1,288 @@
+
+#### **Notes:**
+
+DNF modularity is deprecated, and Rocky Linux does not intend to provide AppStream content as modules in RL 10, although in the future it might be provided as RPM packages, software collections, or Flatpaks.
+
+### Desktop Environment
+
+In RL 10, Wayland replaces the X.Org Server and xorg-x11-server package. Xwayland will support most X11 clients not yet ported to Wayland. Some desktop applications and components are also replaced in RL 10.
+
+**Desktop Components changes**
+
+| Removed | Replacement |
+| :---- | :---- |
+| gedit text editor | GNOME Text Editor (gnome-text-editor) |
+| Tweaks application | Use Tweaks options in GNOME settings |
+| Festival speech synthesizer | Espeak NG |
+| Eye of GNOME (eog) image viewer | GNOME Image Viewer (loupe) |
+| Cheese camera application | Snapshot application |
+| GNOME Terminal | ptyxis terminal |
+| PulseAudio daemon (but not the client interfaces) | Pipewire (which also supports an implementation of the PulseAudio client APIs); Pipewire is the default in RL 9 |
+
+A number of graphical desktop applications are no longer provided by Rocky Linux as part of the RL 10 distribution. In most cases, these applications can be obtained from upstream sources or they might be available as Flatpaks.
+
+**Applications Removed from RL 10**
+
+| Removed | Substitution |
+| :---- | :---- |
+| LibreOffice office productivity software suite | The Document Foundation provides an official Flatpak at Flathub, or you can download upstream packages from [https://www.libreoffice.org/](https://www.libreoffice.org/). |
+| Evolution personal information manager and email client | Thunderbird is available as a Flatpak from the Rocky Linux Ecosystem Catalog, or you can obtain a Flatpak or package from upstream or through Flathub. |
+| Inkscape vector graphics editor | There is no vector graphics editor shipped in RL 10. |
+| Totem (GNOME Videos) media player | There is no default desktop media player shipped in RL 10. The web browser can still play web-based media. |
+
+TigerVNC is removed from RL 10. For VNC client functionality, use the Connections application. No VNC server is provided; gnome-remote-desktop, using RDP, replaces TigerVNC for desktop sharing, remote login, and headless sessions. 
+
+The Motif X11 desktop environment and toolkit are also removed from RL 10.
+
+### Infrastructure Services
+
+Some of the services and packages that are replaced or removed in RL 10 include the following infrastructure services.
+
+** Infrastructure Services changes**
+
+| Removed | Replacement |
+| :---- | :---- |
+| power-profiles-daemon CPU throttle | tuned-ppd translation layer that supports applications written for the power-profiles-daemon |
+| redis in-memory key-value database | valkey in-memory key-value database |
```

Comment:
I'm not sure if everyone followed that drama, might be good to mention that this is a redis fork